examples/task_schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$defs": {
+        "ExampleTaskParameters": {
+            "additionalProperties": true,
+            "description": "Example Task Parameters",
+            "properties": {
+                "field_1": {
+                    "default": 0,
+                    "minimum": 0,
+                    "title": "Field 1",
+                    "type": "integer"
+                },
+                "field_2": {
+                    "default": 0,
+                    "minimum": 0,
+                    "title": "Field 2",
+                    "type": "integer"
+                },
+                "field_3": {
+                    "default": 0.5,
+                    "maximum": 1.0,
+                    "minimum": 0.0,
+                    "title": "Field 3",
+                    "type": "number"
+                },
+                "field_4": {
+                    "default": 0.5,
+                    "maximum": 1.0,
+                    "minimum": 0.0,
+                    "title": "Field 4",
+                    "type": "number"
+                },
+                "field_5": {
+                    "const": "Immutable Field",
+                    "default": "Immutable Field",
+                    "title": "Field 5"
+                }
+            },
+            "title": "ExampleTaskParameters",
+            "type": "object"
+        }
+    },
+    "additionalProperties": false,
+    "description": "Example Task",
+    "properties": {
+        "name": {
+            "const": "TaskName",
+            "default": "TaskName",
+            "title": "Name"
+        },
+        "description": {
+            "default": "Example description of task",
+            "title": "Description",
+            "type": "string"
+        },
+        "version": {
+            "default": "0.0.0",
+            "title": "Version",
+            "type": "string"
+        },
+        "task_parameters": {
+            "allOf": [
+                {
+                    "$ref": "#/$defs/ExampleTaskParameters"
+                }
+            ],
+            "description": "\n    Example Task Parameters\n    "
```

Comment:
Will these escape characters ever be a problem? Should we just add a helper method to strip the NL?